### PR TITLE
Telegram

### PR DESCRIPTION
**Pull request by marcelcorso**
A friend of mine said that they use telegram for monitoring/alerting on their org. I thought it was fun and did it.

(branched from https://github.com/messagebird/sachet/pull/25)

### CODE REVIEW

File: provider/telegram/telegram.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package telegram
+
+import (
+	"strconv"
+
+	"github.com/messagebird/sachet"
+	"gopkg.in/telegram-bot-api.v4"
```
Comment (samwierema) [inline provider/telegram/telegram.go]: Why not use https://github.com/go-telegram-bot-api/telegram-bot-api directly?

Comment (marcelcorso) [inline provider/telegram/telegram.go]: the readme sez: "Use github.com/go-telegram-bot-api/telegram-bot-api for the latest version, or use gopkg.in/telegram-bot-api.v4 for the stable build."

I think "stable" is enough for sachet.

**Comment by rivabem** [comment]
Hey @marcelcorso , now I fell guilty of all those future telegram alerts! :innocent:

**Comment by homelessnessbo** [comment]
To be reviewed after  #25 is merged

**Pull request merged**